docs/governance/community-guidelines/meetings.md

```diff
@@ -0,0 +1,56 @@
+# Communication
+
+This document is a [community guideline].
+
+## Chairing Meetings
+
+It is the responsiblity of the [WG] lead to chair the meetings for their [WG].
+
+In the event the [WG] lead is unable to attend the meeting they should appoint an appropriate delegate from within the [WG] to chair the meeting in their absence. If there are no appropriate delegates for the [WG] then the [FINOS Point of Contact] can be contacted to see if they can chair the meeting.
+
+In the event there are no suitable people to chair the meeting then the meeting should be cancelled.
+
+### Responsibilities of the Chair
+
+The Chair of the meeting is responsible for:
+
+- Opening the Zoom Call: Start the meeting and admit participants.
+- Minutes and Attendance: Ensure someone is taking minutes and that attendees log their attendance by commenting on the issue.
+- Agenda Adherence: Ensure the meeting follows the agenda.
+- Closing the Meeting: Conclude the meeting.
+
+## Meeting Agendas
+
+### Agenda Items
+
+The [WG] lead is responsible for creating an agenda for each meeting. The [WG] lead can seek input from other [WG] members via the mailing list. Alternatively, members can proactively suggest items by reaching out to the [WG] lead.
+
+### Creating the agenda
+
+To create a meeting agenda you can create a new issue in the CCC project on GitHub and selecting the `Common Cloud Controls Meeting Minutes` template.
+
+Update the issue with the correct date/time and add the [WG] name to the end of the issue title.
+
+Update the issue with the correct zoom details.
+
+Fill in the agenda items for the meeting, taking care not to delete any of the items predefined by the template.
+
+Add the `Meeting` label to the issue.
+
+## Meeting Minutes
+
+Minutes can be added as a comment to the corresponding GitHub issue.
+
+Consider circulating the minutes via the mailing list to increase visibility within the [WG].
+
+## Post Meeting Housekeeping
+
+Once minutes are added to the GitHub issue, close the issue.
```

Comment:
Should the issue be closed after the meeting minutes have been approved by the WG on the next meeting or as soon as minutes posted to the issue as a comment?

Reply:
I'm happy with whatever route is the best presentation for leads, members, and new joiners. Seems Stevie's stance is to close the issue quickly so that we don't have issues left open for 2 weeks+.

cc @robmoffat @Alexstpierrework

Reply:
The correct approach is to review and approve the meeting minutes from the last meeting in the current one, and then close it at that point.  However, I've never really seen that done it's a bit laborious.  

@eddie-knight you created a GitHub action that uh, (checks note) closes all issues after a month anyway, right ?  https://github.com/finos/common-cloud-controls/pull/205/files#diff-b5e0854ed0838024d5cc25b4e030922e34648b9ea8c432807e35495fb805b894

Reply:
Yep. 

Actually we could add a stage specifically for issues tagged as meetings, if we want. We could have them auto-close after 3 weeks if they're left open (since we have 3 weeks between calls in longer months).

This could be a precaution if we keep the required step of closing them on the next call.